.github/workflows/release.yml

```
@@ -133,16 +125,25 @@ jobs:
           echo ".idea" >> .gitignore
           echo ".ultra.cache.json" >> .gitignore
           echo "db/*" >> .gitignore
-        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
+        if: ${{ steps.check-artifact.outputs.exists != 'true' }}
 
       - name: Get modified files
         id: diff
-        run: git ls-files --others --exclude-standard >> artifacts_list.txt
-        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)